docs/howtoguides/enable_landscape.rst

```diff
@@ -0,0 +1,47 @@
+How to enable Landscape
+***********************
+
+``pro`` offers a convenient way to register a machine with Landscape via the
+``pro enable landscape`` command.
+
+Enable interactively
+====================
+
+To register your machine by interactively providing your Landscape account
+details at the CLI, run:
+
+.. code:: bash
+
+   sudo pro enable landscape
+
+That command will install ``landscape-client`` and drop you into an interactive
+wizard to complete the Landscape registration for the machine.
+
+Enable non-interactively
+========================
+
+If you know the details of your Landscape setup and would like to register a
+machine without any interaction necessary, you can pass ``landscape-config``
```

Comment:
```suggestion
machine without using the wizard, you can pass ``landscape-config``
```

Reply:
Also, a question a reader might naturally have is "what's landscape-config and do I need to install it?". It's probably worth mentioning that it's part of the pro enable landscape command.

Reply:
I'll make the suggested change.

`landscape-config` isn't part of `pro enable landscape` per se. It is a command installed as part of the `landscape-client` package and owned by the landscape team. It could completely change underneath of us without us knowing, so we shouldn't try to document it too much. The only web-accessible documentation that I can find (which I would've linked before but didn't find it until now), is the manpage on line here: https://manpages.ubuntu.com/landscape-config

So I'll add that link and an explanation that under-the-hood, pro enable landscape installs landscape-config and executes it, which should answer the "do I need to install it" part